src/main/java/com/snowflake/kafka/connector/internal/SnowflakeConnectionService.java

```diff
@@ -106,9 +107,9 @@ public interface SnowflakeConnectionService {
    * Alter table to add columns according to a map from columnNames to their types
    *
    * @param tableName the name of the table
-   * @param columnToType the mapping from the columnNames to their types
+   * @param columnInfos the mapping from the columnNames to their types and comments
    */
-  void appendColumnsToTable(String tableName, Map<String, String> columnToType);
+  void appendColumnsToTable(String tableName, Map<String, Pair<String, String>> columnInfos);
```

Comment:
It would be better to introduce a separate class instead of `Pair` here to explain what we're passing between objects.

Reply:
i fix it to use class ColumnInfos